python/mxnet/contrib/onnx/__init__.py

```diff
@@ -18,3 +18,4 @@
 
 from ._import.import_model import import_model, get_model_metadata
```

Comment:
this does not make sense to me.
why do you want to put public function in private module folder `_import` or `_export` and include them later?

Reply:
``import`` is a reserved keyword, we cant have a folder called import. we can probably rename the two folders to onnx_import and onnx_export and make its member files private, except for the modules that we are exposing to the user.

Reply:
folder name changed to be public ,  _import --> onnx2mx , _export-->mx2onnx . also changed the files in the folder as per their usage